spec/comment_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  subject { Comment.new(text: 'Nice post', user_id: 1, post_id: 1) }
+
+  before { subject.save }
+
+  it 'new comment should be saved in the database' do
+    expect(subject.new_record?).to be_truthy
```

Comment:
## * Kindly Add unit specs for all of your models' methods as unit tests for them are missing ( for example you have the method `update_post_comments_counter` which needs to be tested ) 
![image](https://github.com/torobucci/blog_app/assets/52497246/8955219d-0c4f-41dc-bf1a-ed6ade4df6db)